ndk/src/data_space.rs

```diff
@@ -5,13 +5,13 @@
 
 use std::fmt;
 
-use num_enum::{IntoPrimitive, TryFromPrimitive, TryFromPrimitiveError};
+use num_enum::{FromPrimitive, IntoPrimitive};
 
 /// Describes how to interpret colors.
 ///
 /// <https://developer.android.com/ndk/reference/group/a-data-space#group___a_data_space_1ga2759ad19cae46646cc5f7002758c4a1c>
-#[repr(u32)]
-#[derive(Clone, Copy, Hash, PartialEq, Eq, TryFromPrimitive, IntoPrimitive)]
+#[repr(i32)]
+#[derive(Clone, Copy, Hash, PartialEq, Eq, FromPrimitive, IntoPrimitive)]
 #[doc(alias = "ADataSpace")]
 #[non_exhaustive]
 pub enum DataSpace {
```

Comment:
Note that the native type here will change to `i32`:

https://android-review.googlesource.com/c/platform/frameworks/native/+/2902242